sample/Aguacongas.AspNetCore.Authentication.Sample/Controllers/HomeController.cs

```diff
@@ -133,12 +141,12 @@ public async Task<IActionResult> Update(AuthenticationViewModel model)
         [Route("List")]
         public async Task<IActionResult> List()
         {
-            var schemes = await _signInManager.GetExternalAuthenticationSchemesAsync();
+            System.Collections.Generic.IEnumerable<AuthenticationScheme> schemes = await _signInManager.GetExternalAuthenticationSchemesAsync();
 
-            var managedSchemes = schemes.Where(s => _manager.ManagedHandlerType.Any(h => s.HandlerType == h))
+            System.Collections.Generic.IEnumerable<string> managedSchemes = schemes.Where(s => _handerTypeProvider.GetManagedHandlerTypes().Any(h => s.HandlerType == h))
                 .Select(s => s.Name);
 
-            var definitions = managedSchemes.Select(name => _manager.FindBySchemeAsync(name).GetAwaiter().GetResult());
+            System.Collections.Generic.IEnumerable<SchemeDefinition> definitions = managedSchemes.Select(name => _mutationStore.FindBySchemeAsync(name).GetAwaiter().GetResult());
```

Comment:
can you simplify by adding a using System.Collections.Generic or using var please